Nitrox.Model/Helper/NetHelper.cs

```diff
@@ -182,8 +228,14 @@ static long GetTotalInternetBytes() =>
     /// </summary>
     public static bool IsPrivate(this IPAddress address)
     {
+        address = NormalizeAddress(address);
```

Comment:
I'm not sure what the benefit of doing this at a high scope is, can we not move this with the range method below as the other range method calls it itself anyway